src/Stream/test/StreamBase.Test/Extensions/HostBuilderExtensionsTest.cs

```diff
@@ -13,19 +16,13 @@
 
 namespace Steeltoe.Stream.Extensions
 {
-    public class HostBuilderExtensions
+    public class HostBuilderExtensionsTest
     {
         [Fact]
-        public void AddStreamsServices_AddsServices()
+        public void HostBuilderExtensionTest()
         {
-            var hostBuilder = Host.CreateDefaultBuilder().
-                  AddStreamsServices<SampleSink>();
-            var server = hostBuilder
-                .AddStreamsServices<SampleSink>()
-                .Build();
-
-            var service = server.Services.GetService<IHostedService>();
-            Assert.NotNull(service);
+            var hostBuilder = Host.CreateDefaultBuilder().AddStreamsServices<SampleSink>();
+            Assert.NotNull(hostBuilder);
```

Comment:
This test should probably assert that `AddStreamsServices<T>` did something (besides not throw an exception)